### PR TITLE
[Term Entry] PHP Math Functions is_infinite()

### PR DESCRIPTION
**Pull request by LewisMK**
### Description

PHP is_infinite() function. Closes issue #2400 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/math-functions/terms/is-infinite/is-infinite.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  $result = is_infinite($value);
+  echo $result; //this will output '1', indicating True
+?>
+```
+
+## Codebyte Example
+
+The codebyte below deploys the `is_infinite()` function on the values of `100` and `log(0)` to evaluate whether they are `infinite`:
+
+```codebyte/php
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/is-infinite/is-infinite.md:44]: code bytes are optional. Ideally all the examples included in an entry are serving a purpose and are not duplicative. This codebyte, is in part, restating the content from the previous example. I would suggest revising the example or deleting it.

**Review by caupolicandiaz — APPROVED** [review]
@LewisMK Looks good. I just pushed some spacing edits. I'm passing this along for a second review.

**Comment by LewisMK** [comment]
@caupolicandiaz thank you for the guidance.

**Review by SSwiniarski — APPROVED** [review]
@LewisMK looks good.

**Comment by SSwiniarski** [comment]
@LewisMK Congrats on the new entry! Live link: https://www.codecademy.com/resources/docs/php/math-functions/is-infinite

image

**Comment by LewisMK** [comment]
@SSwiniarski thanks so much. I am so excited seeing the live link. I will be contributing more in future.

**Pull request merged**